e2e/config/config.go

```diff
@@ -12,9 +12,13 @@ import (
 )
 
 const (
-	defaultChannelNamespace = "e2e-gitops"
-	defaultGitURL           = "https://github.com/RamenDR/ocm-ramen-samples.git"
-	defaultGitBranch        = "main"
+	defaultChannelNamespace   = "e2e-gitops"
+	defaultGitURL             = "https://github.com/RamenDR/ocm-ramen-samples.git"
+	defaultGitBranch          = "main"
+	defaultRamenHubNamespace  = "ramen-system"
+	defaultDRClusterNamespace = "ramen-system"
+	defaultRamenopsNamespace  = "ramen-ops"
+	defaultArgocdNamespace    = "argocd"
```

Comment:
This constants list is becoming little bit too long, maybe separate it by config sections?

```go
const (
    // Repo
    defaultGitURL = "..."
    defaultGitBranch = "..."

    // Namspaces
    ...

    // Channel
    ...
```